.gitmodules

```diff
@@ -177,7 +177,7 @@
 [submodule "helix-syntax/languages/tree-sitter-llvm-mir"]
 	path = helix-syntax/languages/tree-sitter-llvm-mir
 	url = https://github.com/Flakebi/tree-sitter-llvm-mir.git
-  shallow = true
+    shallow = true
```

Comment:
Please note this addition, introduced intentionally while resolving a merge conflict